docker-compose.yaml

```diff
@@ -13,29 +12,23 @@ services:
       POSTGRES_USERNAME: postgres
       POSTGRES_HOST: ffc-demo-user-postgres
       POSTGRES_PORT: 5432
-    networks:
-      - ffc-demo-users
-    ports:
-      - "3002:3002"
     volumes:
       - node_modules:/usr/src/app/node_modules
       - .:/usr/src/app
+    networks:

```

Comment:
Nitpick: no empty line between `volumes` and `networks` blocks. Also, the order of these blocks has reversed compared to other docker-compose files.

Reply:
I've swapped the blocks over. We appear to only have an empty line in the final volume and network definitions in the file, rather than within the service definition in other repos, so have not added a blank line.

Reply:
Right you are, I misread it in the GitHub diff view and thought this was the network definition at the end.